yt/python/yt/wrapper/cli_impl.py

```diff
@@ -1,5 +1,9 @@
 import yt.wrapper as yt
 import yt.logger as logger
+from .http_helpers import get_user_name
+from .common import YtError
+from .auth_commands import encode_sha256
+from getpass import getpass
```

Comment:
We usually order imports from local one to global one and separate groups within newline, in this case:
```
from .http_helpers import get_user_name
from .common import YtError
from .auth_commands import encode_sha256

import yt.wrapper as yt
import yt.logger as logger

from getpass import getpass
```

Reply:
Done